### PR TITLE
contribute.json

### PR DESCRIPTION
**Pull request by peterbe**
This is just a start to tackle https://github.com/mozilla-services/buildhub/issues/319

Once this is reviewed, we need to get it into `gh-pages` somehow.

### CODE REVIEW

File: contribute.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "name": "buildhub",
+    "description": "Firefox Build Metadata Service",
```
Comment (leplatrem) [inline contribute.json]: I would put *Mozilla Product* instead of just Firefox

File: contribute.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "participate": {
+        "irc": "irc://irc.mozilla.org/#storage-team",
```
Comment (leplatrem) [inline contribute.json]: we have a `#product-delivery` channel, I think we should agree on using only one /cc @mostlygeek

Comment (peterbe) [reply contribute.json]: So what should I put?

Comment (leplatrem) [reply contribute.json]: Leave it like this I guess

Comment (peterbe) [reply contribute.json]: left it as is :)

**Pull request merged**